thunx-encoding-json/src/main/java/eu/xenit/contentcloud/thunx/encoding/json/ExpressionJsonConverter.java

```diff
@@ -75,6 +76,10 @@ public JsonExpressionDto visit(Variable variable) {
             return new JsonVariableDto(variable.getName());
         }
 
+        @Override
+        public JsonExpressionDto visit(ResolvedExpression<?> resolvedExpression) {
+            throw new IllegalArgumentException("Resolved expression is not supported.");
```

Comment:
In my opinion, the encoder is not the correct place to contain the logic that resolved expressions should  probably not be encoded as json, in our particular use case ?